### DISCUSSION TITLE
go2rtc showing 0 online streams with default settings

### DISCUSSION THREAD

**Discussion by vladcooks** [gtxaspec/wz_mini_hacks#643] [Q&A]
I was wondering if I am missing a step because I've been trying to figure this out for a while now. The "default" rtsp server works fine for me but when I enable go2rtc in the config and try to use it my camera is showing offline for both streams. 

I just did a clean install of wz_mini to reset all settings to default and only enabled the go2rtc related ones. I have V3 cams and from what I can see in the config under localhost:1984 its set to the correct output by default

![image](https://github.com/gtxaspec/wz_mini_hacks/assets/99919705/76d432d9-d488-4e48-8b27-1f054d29eaef)

**Comment by marcousa** [comment]
Did you ever solve this?
I’m having the same issue and can’t figure out how to get a stream to work.

**Comment by vladcooks** [comment]
I was running an unsupported firmware. I had to downgrade to a supported
version
